Systems/c172p-damage.xml

```diff
@@ -243,38 +243,39 @@
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-high
             </test>
+            
             <test logic="AND" value="1.0">
                 /velocities/airspeed-kt GT damage/limits-vne-medium
                 damage/roll-moment GT 4000.0
-                wing-damage/right-wing GT 0.0
+                wing-damage/right-wing LT 1.0
             </test>
 
             <!-- Over-g -->
             <test logic="AND" value="1.0">
                 damage/force-aero-lbs GT damage/limits-lift-high
             </test>
+            
```

Comment:
It was to make it easier to read. In a first step, I didn't see the two `< test>` blocks.
But that's formal, not important.
Just tell me your preference.